components/DropdownLight/DropdownLight.unit.test.jsx

```diff
@@ -225,4 +230,19 @@ describe('<DropdownLight />', () => {
       placeholderContent,
     );
   });
+
+  it('should show if there is icon at item', () => {
+    render(<DropdownLight items={itemsWithImageMock} />);
+
+    const dropdown = screen.getByRole('button');
+    userEvent.click(dropdown);
+
+    const optionItem = screen.getByRole('option', {
+      name: 'Visa image description',
+    });
+
+    const selectionItemImage = optionItem.querySelector('img');
```

Comment:
i think this querySelector is not necessary.
```
  it('should show if there is icon at item', () => {
    render(<DropdownLight items={itemsWithImageMock} />);

    const dropdown = screen.getByRole('button');
    userEvent.click(dropdown);

    const optionItem = screen.getByRole('option', {
      name: 'Visa image description',
    });

    expect(optionItem).toBeInTheDocument();
```

is there any reason for its use?

Reply:
I understand that JsVanilla's native DOM should be avoided for testing. However, in image tests the querySelector would be the most correct, especially if in the future we want to test a specific image.

here are two articles that talk more about the subject:

https://robertmarshall.dev/blog/test-images-in-react-with-jest/#:~:text=The%20Jest%20test%20would%20be,%3E)%20const%20testImage%20%3D%20document

https://medium.com/@drake_beth/how-to-test-images-in-react-a70053b1634a

Reply:
I agree, this logic sounds correct. :+1: